fixture-generator/src/bin/main.rs

```diff
@@ -73,6 +73,13 @@ struct MoveFixture {
     args: [MoveArg; 3],     // vk, public_values, proof
 }
 
+fn bytes(proof: SphinxProofWithPublicValues) -> String {
+    match proof.proof {
+        SphinxProof::Plonk(pr) => pr.raw_proof,
```

Comment:
@storojs72: Can you double check this part of the code? I'm not sure, but I think this might need to be `pr.encoded_proof` instead of `pr.raw_proof`. Specifically, the `encoded_proof` is returned by gnark [here](https://github.com/succinctlabs/sp1/blob/6a6c06481712eb5362dcb1dbe37a241e932dd690/crates/recursion/gnark-ffi/go/sp1/utils.go#L25) as the result of calling `MarshalSolidity`

This may also require an update to `sphinx-contracts`